apps/src/util/experiments.js

```diff
@@ -39,7 +43,10 @@ experiments.setEnabled = function (key, shouldEnable) {
   } else {
     return;
   }
-  localStorage.setItem(STORAGE_KEY, JSON.stringify(allEnabled));
+  try {
+    localStorage.setItem(STORAGE_KEY, JSON.stringify(allEnabled));
+  } catch (e) {
+  }
```

Comment:
It seems we've run into this before... maybe just use [`utils.trySetLocalStorage()`](https://github.com/code-dot-org/code-dot-org/blob/70d91d076903dfbd8538e6190b01a6fe718e4ee6/apps/src/utils.js#L418-L418)?